tests/python/kaolin/io/test_shapenet.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from pathlib import Path
+
+import os
+
+import pytest
+import torch
+
+from kaolin.io.obj import return_type
+from kaolin.io.shapenet import ShapeNet
+
+SHAPENET_PATH = 'data/ShapeNet'
```

Comment:
I think we should assume `/data/ShapeNet`, `data/ShapeNet` is a relative path and should avoided.

Reply:
Ah, I was trying to run some tests locally. Changed now.